app/controllers/application_controller.rb

```diff
@@ -1,5 +1,14 @@
 class ApplicationController < ActionController::Base
-  def current_user
-    @current_user = User.first
+  protect_from_forgery with: :exception
+  before_action :configure_permitted_parameters, if: :devise_controller?
+  before_action :authenticate_user!
+  # load_and_authorize_resource
+
```

Comment:
- [ ] It would be nice if you remove the commented code to make your code follow [Ruby best practices](https://github.com/microverseinc/curriculum-ruby/blob/main/articles/ruby_best_practices.md) 👍